kdbg/dbgdriver.cpp

```diff
@@ -296,7 +296,7 @@ void DebuggerDriver::processOutput(const QByteArray& data)
 	m_logFile.write(data);
 	m_logFile.flush();
     }
-    
+
```

Comment:
Please do not include whitespace changes that are far away from the necessary changes.